documentation/building/linux-instructions.md

```diff
@@ -1,210 +1,57 @@
-Linux Prerequisites 
+Linux Prerequisites
 ===================
 
 These instructions will lead you through preparing to build and test the diagnostics repo on Linux. We'll start by showing how to set up your environment from scratch. In some cases, a version lldb/llvm that works the best on the distro will have to be built and installed.
 
 Toolchain Setup
 ---------------
 
-The following instructions will install the required packages. This only needs to be done once per machine. These instructions assume that you already have "sudo" installed. It is also recommended to create a github fork of the diagnostics repo and cloning that instead of https://github.com/dotnet/diagnostics.git directly.
+SOS plugin requires lldb v3.4 and above. It is recommeneded to use latest version of llvm toolchain, but any version >= 3.4 should work.
```

Comment:
It needs to be 3.9 or above not 3.4.

Reply:
I skimmed through SOS plugin usage of lldb APIs, and inferred that we are compatible with 3.4 and above. Must have missed something which require minimum 3.9. Will update.

```suggestion
SOS plugin requires lldb v3.9 and above. It is recommended to use latest version of llvm toolchain, but any version >= 3.9 should work.
```